File: app/src/main/java/com/example/androidtest/utils/networkutils/APIEndpoint.java
package com.example.androidtest.utils.networkutils;

import com.example.androidtest.model.APIResponse;
import retrofit2.Call;

import retrofit2.http.GET;

public interface APIEndpoint {
    @GET("s/2iodh4vg0eortkl/facts.json")
    Call<APIResponse> getResponse();
}

File: app/src/main/java/com/example/androidtest/utils/networkutils/APIEndpoint.kt
package com.example.androidtest.utils.networkutils

import com.example.androidtest.model.APIResponse
import retrofit2.Call
import retrofit2.Response

import retrofit2.http.GET

interface APIEndpoint {
    @GET("s/2iodh4vg0eortkl/facts.json")
    suspend fun response(): Response<APIResponse>
}
